Source/GSXib5KeyedUnarchiver.m

```diff
@@ -449,7 +449,8 @@ - (NSString*) alternateName: (NSString*)name
   NSString *postfix = [self alternateName: name startIndex: 2];
   NSString *typeName = [currentElement attributeForKey: @"key"];
 
-  if ((typeName != nil) && [postfix hasPrefix: typeName])
+  if ((typeName != nil) && [postfix hasPrefix: typeName] &&
```

Comment:
Why is this part of your pull request? This is already on the master branch.

Reply:
I had backed out some changes on my branch.  I did so by copying the file from master, I'll make sure this doesn't accidently back anything out on master.